client/src/components/Notary/NotarizationTable.tsx

```diff
@@ -87,12 +87,15 @@ export const NotarizationTable: React.FC<NotarizationTableProps> = ({
 
   const [currentPage, setCurrentPage] = useState(0);
 
+  const [isInstallExtensionClicked, setIsInstallExtensionClicked] = useState<boolean>(false);
+
   /*
    * Handlers
    */
 
   const handleInstallExtensionClicked = () => {
     alert('This will link to extension download page');
```

Comment:
I think you need to rebase, I updated this to `window.open(CHROME_EXTENSION_URL, '_blank');`